client/src/components/pages/application/Project/index.js

```diff
@@ -1,5 +1,135 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import { TextareaAutosize } from '@material-ui/core';
+import TextField from '../../../common/TextField';
+import Limitation from '../../../common/limitation';
+import Typography from '../../../common/Typography';
+import backGround from '../../../../assets/images/backGround.svg';
+import Button from '../../../common/Button';
+import FProjectValidation from '../../../../utils/application/ProjectValidation';
 
-const Project = () => <h1>Project</h1>;
+import './style.css';
+
+const getUserID = async () => {
+  const { data } = await axios.get('/api/v1/isUser');
+  return data;
+};
+const projectData = async (ID) => {
+  const {
+    data: { project },
+  } = await axios.get(`/api/v1/project/${ID}`);
+  return project;
+};
+const Project = () => {
+  const [projectTitle, setProjectTitle] = useState('');
+  const [projectDesc, setProjectDesc] = useState('');
+  const [githubLink, setGithubLink] = useState('');
+  const [message, setMessage] = useState([]);
+  const [UserId, setId] = useState('');
+  const history = useHistory();
+
+  const throwMessage = (msg) => setMessage(msg);
+
+  useEffect(() => {
+    getUserID().then((data) => {
+      if (data.message === 'you are authorized') {
+        setId(data.userId);
+      }
+    });
+    if (UserId) {
+      axios.get(`/api/v1/applicants/${UserId}`).then(({ data: { user } }) => {
+        setGithubLink(user.githubLink);
```

Comment:
it will not update on `githubLink` but in `projectGithubLink`, just focus on that and update it .
this link is only for Final project Link on github not for github itself, that was in Accounts. 
For more: look at applicants module at `projectGithubLink` field. 
```suggestion
        setGithubLink(user.projectGithubLink);
```